src/popup/stores/SessionStore.ts

```diff
@@ -9,13 +9,15 @@ const INIT_VALUES = {
   loggedInAccount: undefined,
   info: undefined,
   qtumBalanceUSD: undefined,
+  sessionLogoutInterval: undefined,
 };
 
 export default class SessionStore {
   @observable public networkIndex: number = INIT_VALUES.networkIndex;
   @observable public loggedInAccount?: Account = INIT_VALUES.loggedInAccount;
   @observable public info?: Insight.IGetInfo = INIT_VALUES.info;
   @observable public qtumBalanceUSD?: string = undefined;
+  @observable public sessionLogoutInterval: number = 0; // this value will be overridden on component load by sessionBackground thru MESSAGE_TYPE.GET_SESSION_LOGOUT_INTERVAL
```

Comment:
guessing this needs to be 0 when you first load the component. in that case, make the `INIT_VALUES.sessionLogoutInterval: 0` and use that here.